<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="toolButtonLightA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Light-A</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonLightB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Light-B</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Light Level</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Light Level</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxLightA">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxLightB">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReflectLight">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>239</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reflect Light</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QCheckBox" name="checkBoxIn0">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>239</red>
           <green>239</green>
           <blue>239</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>In-0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxIn1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>239</red>
           <green>239</green>
           <blue>239</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>In-1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxIn2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>239</red>
           <green>239</green>
           <blue>239</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>In-2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxIn3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>239</red>
           <green>239</green>
           <blue>239</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>In-3</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QCheckBox" name="checkBoxOut0">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Out-0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxOut1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Out-1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxOut2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Out-2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxOut3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Out-3</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Camera I/O</string>
    </property>
    <widget class="QCheckBox" name="checkBoxCamIN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>239</red>
           <green>239</green>
           <blue>239</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>Cam-IN</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxCamOut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Cam-Out</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonReflectIO">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>320</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reflect I/O</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
